keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -2788,79 +2894,76 @@ previous answers based on experience in the field.
 
 ###### Will enabling / using this feature result in any new API calls?
 
-<!--
-Describe them, providing:
-  - API call type (e.g. PATCH pods)
-  - estimated throughput
-  - originating component(s) (e.g. Kubelet, Feature-X-controller)
-Focusing mostly on:
-  - components listing and/or watching resources they didn't before
-  - API calls that may be triggered by changes of some Kubernetes resources
-    (e.g. update of object X triggers new updates of object Y)
-  - periodic API calls to reconcile state (e.g. periodic fetching state,
-    heartbeats, leader election, etc.)
--->
+For Pods not using ResourceClaims, not much changes. The following components
+need to watch additional resources:
+- kube-controller-manager: ResourceClaimTemplate
+- kube-scheduler: ResourceClaim, DeviceClass, ResourceSlice
+- kubelet: ResourceClaim
```

Comment:
presumably only the ones relevant for that node?

Reply:
Actually, `kubelet: ResourceClaim` was wrong. We replaced that with doing a GET precisely because watching all by all kubelets would have been a scalability problem. The GET adds latency, but it scales linearly with number of nodes and claims.

Removed.